xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/displayers/linkDisplayer.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+define([
+  BASE_PATH + "displayers/defaultDisplayer.js"
+], function (
+  DefaultDisplayer
+) {
+
+
+  /**
+   * Create an HTML displayer for a property of an entry
+   * Extends the default displayer class
+   */
+  var LinkDisplayer = function (propertyId, entryData, logic) {
+    DefaultDisplayer.call(this, propertyId, entryData, logic);
+  };
+  LinkDisplayer.prototype = Object.create(DefaultDisplayer.prototype);
+  LinkDisplayer.prototype.constructor = LinkDisplayer;
+
+
+  /**
+   * Create link viewer element for the displayer
+   * Override the default inherted method
+   */
+  LinkDisplayer.prototype.createView = function (defer, params) {
+    // default href
+    var propertyHref = params.config.propertyHref && params.entry[params.config.propertyHref];
+    var href = propertyHref || params.config.href;
```

Comment:
I'm not sure it makes sense to have a default URL specified in the displayer configuration. You need to be aware that any feature you add it's hard to remove afterwards (even if in practice it's not used) because we cannot break backwards compatibility. So we need to be careful with what we expose as **public** API or configuration.